tutorial/numpy-functions.md

```diff
@@ -0,0 +1,351 @@
+# Supported Numpy function by Pyccel
+
+In Pyccel we try to support the most used Numpy functions by developers. Here are some of them:
```

Comment:
Better grammar:
`In Pyccel we try to support the Numpy functions which developers use the most.`